src/styles/colors/colors.ts

```diff
@@ -23,6 +23,8 @@ export const COLORS = {
   textColor2: "#1C1939CC",
```

Comment:
@Allen-Muhani @Chegemartinx @Alphonce-Mutebi 
I cant see a color called textColor2 on Figma.

Why is it so difficult to make these things match?
This should be very easy. not rocket science. Where is the problem?

We are repeating ourselves. This is not really a coding problem, but an organizational one. how can we fix that?

Reply:
![text color screen shot](https://user-images.githubusercontent.com/34819411/159873577-dcf00e1d-1f87-4da2-8442-3895b5bb0e8c.png)
Here it is